src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -133,12 +133,21 @@ variant_add_commit_details (GVariantDict *dict,
 {
   g_autoptr(GVariant) metadata = NULL;
   g_autofree gchar *version_commit = NULL;
+  g_autofree gchar *end_of_life = NULL;
   guint64 timestamp = 0;
 
   timestamp = ostree_commit_get_timestamp (commit);
   metadata = g_variant_get_child_value (commit, 0);
   if (metadata != NULL)
-    g_variant_lookup (metadata, "version", "s", &version_commit);
+    {
+      g_variant_lookup (metadata, "version", "s", &version_commit);
+      /* avoid double insertion */
+      if (prefix != NULL)
+        g_variant_lookup(metadata, "end-of-life", "s", &end_of_life);
```

Comment:
`Also, for ostree metadata there's a convention to use namespace prefixes`

So in order to match the namespace prefixes, it will be 'g_variant_lookup(metadata, "ostree.end-of-life", "s", &end_of_life)' instead? 


`I think we should have this metadata defined in ostree, so it'd be ostree.end-of-life. In fact...this is even an example in ostree today:`

I was thinking earlier that end-of-life attribute may show up in metadata only if OS vendor add it with the commit like 
'ostree commit --add-detached-metadata-string="endoflife =" ' , so each commit will potentially not have end-of-life attribute at all. I am assuming if we have this metadata defined in ostree, we would need to include it in every ostree commit?